src/controllers/webhookController.ts

```
@@ -398,6 +398,88 @@ export async function handleFlutterwaveWebhook(
+    });
+    res.status(401).json({ error: "Webhook timestamp invalid or expired" });
+    return;
+  }
+
+  const received = req.headers["x-bills-signature"] as string | undefined;
+  if (!received) {
+    throw new AppError(
+      "Missing x-bills-signature header",
+      401,
+      ErrorCodes.MISSING_SIGNATURE,
+    );
```

ESLint

[error] Replace `⏎······"Missing·x-bills-signature·header",⏎······401,⏎······ErrorCodes.MISSING_SIGNATURE,⏎····` with `"Missing·x-bills-signature·header",·401,·ErrorCodes.MISSING_SIGNATURE`

(prettier/prettier)